packages/core/src/index.spec.ts

```diff
@@ -0,0 +1,41 @@
+import { debug, expandVars, HoprDB, PublicKey } from '@hoprnet/hopr-utils'
+import HoprCoreEthereum from '@hoprnet/hopr-core-ethereum'
+import { expect } from 'chai'
+import PeerId from 'peer-id'
+import sinon from 'sinon'
+import Hopr, { HoprOptions, resolveEnvironment, VERSION } from '.'
+
+const log = debug('hopr-core:test:index')
+
+describe('hopr controller ', async function () {
+  let peerId: PeerId, db: HoprDB, chain: HoprCoreEthereum, options: HoprOptions
+  beforeEach(async function () {
+    peerId = await PeerId.create({ keyType: 'secp256k1', bits: 256 })
+    options = new HoprOptions(resolveEnvironment('master-xdai'))
```

Comment:
We should not use a production environment in tests, even if it might not be called.  Also I removed the env.

Reply:
Ah, good point, need to mock this one then.

Reply:
Just updated the mock, resolving convo.